### ISSUE TITLE
no stack whatsoever in TUI

### ISSUE THREAD

**Issue by yarikoptic** [P403n1x87/austin#52]
### Description

I only see
```
 Austin  TUI  Memory Profile
  _________   PPID 968515 CMD /home/yoh/proj/misc/austin/venvs/dev3/bin/python /home/yoh/proj/misc/austin/venvs/dev3/bin/datalad install -s /// /tmp/in
stallhere_⎠   PID:TID 968515:7fde87514    1  of   2   CPU     0%     ▁      
              Samples        2  Duration      07"     MEM    34M    ▂███████
  OWN    TOTAL    %OWN   %TOTAL  FUNCTION                                                                                                              
     0       4    0.00%  11.76%  _find_and_load (<frozen importlib._bootstrap>)   
```
on some runs... on some other it stays empty entirely (until finishing up and showing #51 upon exit).

running regular `austin` (`rm -rf /tmp/installhere; austin -C -m /home/yoh/proj/misc/austin/venvs/dev3/bin/datalad install -s /// /tmp/installhere`) produces lots of reasonable output showing a nice flamegraph if passed to 

### Steps to Reproduce

Here is the script
```
#!/bin/bash

PS4='> '
set -xeu

cd "$(mktemp -d ${TMPDIR:-/tmp}/dl-XXXXXXX)"
pwd
# sudo apt-get install git git-annex
git clone https://github.com/P403n1x87/austin

cd austin
py=3; d=venvs/dev$py; virtualenv --python=python$py --system-site-packages $d && source $d/bin/activate && pip3 install -e .
pip install datalad
cd ..
austin-tui -C -l -m austin/venvs/dev3/bin/datalad install ///

```
and interesting part is that on the first run -- it showed some stack... in full mode it became even more sensible... upon reruns -- success varied, and often I just had the `<frozen...` one only.  (BTW -- may be those could be hidden unless in full mode? probably coming up due to virtualenv )

anyways -- most likely related to #51 , I will leave a comment there now with some more details

**Comment by P403n1x87** [comment]
When in normal mode, the TUI displays the last seen stack frame. If you want to see the full data that has been collected during the run, switch to full mode. For any serious profiling though, I recommend using flamegraphs or https://speedscope.app.

**Comment by P403n1x87** [comment]
Potentially fixed by [austin-tui@v0.1.2](https://github.com/P403n1x87/austin-tui/releases/tag/v0.1.2). If not then please re-open.
